File: app/src/main/java/com/moviepass/MoviePosterClickListener.java
package com.moviepass;

import android.widget.ImageView;

import com.moviepass.model.Movie;

/**
 * Created by ryan on 4/27/17.
 */

public interface MoviePosterClickListener {

    void onMoviePosterClick(int pos, Movie movie, ImageView sharedImageView);
}

File: app/src/main/kotlin/com/moviepass/listeners/MoviePosterClickListener.kt
package com.moviepass

import android.widget.ImageView

import com.moviepass.model.Movie

/**
 * Created by ryan on 4/27/17.
 */

interface MoviePosterClickListener {

    fun onMoviePosterClick(pos: Int, movie: Movie, sharedImageView: ImageView)
}
